target/linux/ramips/dts/ApexeraNetBot.dts

```diff
@@ -0,0 +1,183 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "apexera-netbot", "ralink,mt7620a-soc";
+	model = "Apexera NetBot";
+
+	chosen {
+		bootargs = "console=ttyS1,57600";
+        };
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wifi {
+			label = "netbot:led:wifi";
+			gpios = <&gpio2 27 1>;
```

Comment:
Use the GPIO_ACTIVE_LOW and GPIO_ACTIVE_HIGH macros instead of 1 and 0. Check the existing dts files for reference.

Reply:
the macro is really visable, thank you.